### PR TITLE
Domains: Update Incoming Transfer links

### PR DESCRIPTION
**Pull request by aidvu**
Update all links for incoming transfers. Support stubs created.

### CODE REVIEW

File: client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx [hunk 1]
```diff
@@ -68,8 +69,7 @@ class InboundTransferEmailVerificationCard extends React.Component {
 						components: {
 							learnMoreLink: (
 								<a
-									href="http://support.wordpress.com"
-									target="_blank"
+									href={ support.INCOMING_DOMAIN_TRANSFER }
```
Comment (aidvu) [inline client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx]: @danhauk, do we need a different link here?

Comment (danhauk) [reply client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx]: @aidvu We'll have a support article specifically for the various statuses of the transfer. The slug will be `/incoming-domain-transfer/status-and-failed-transfers/`. I'm not sure of the exact hash anchors yet, but the statuses we'll have on that page are:

* Pending Confirmation (and how to resend the email)
* Pending Transfer
* Transfer Failed
* Cancel Transfer

**Comment by aidvu** [comment]
@danhauk IIRC, there was a discussion on this, and it was agreed that we shouldn't open new windows for our own support links. Tho if you think, can make them open in a new window.

I'll update the 2 links and we can add the anchors to these once the articles are written.

**Comment by danhauk** [comment]
@aidvu I'll think some more on that. I agree with the overall idea of not opening new tabs/windows for our own support links. My only concern is the fact that I lose my place in the flow if I click on a support link and have to do it all over again. It could get tiresome if I want to open a support article for each step and forget to open in a new tab.

Let's leave it for now so as not to hold up this PR and we can revisit if we get any feedback about it.

**Comment by aidvu** [comment]
The `in progress` one was already fixed here. Good :eye: noticing the cancelled one. Updated. I'll update again when I get the anchors for statuses.

**Comment by danhauk** [comment]
Ah, sorry about that. Thanks for updating these!

File: client/my-sites/domains/domain-management/components/icann-verification/icann-verification-card.jsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ class IcannVerificationCard extends React.Component {
 			<EmailVerificationCard
 				changeEmailHref={ changeEmailHref }
 				contactEmail={ contactDetails.email }
+				headerText={ translate( 'Important: Verify Your Email Address' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/components/icann-verification/icann-verification-card.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Verify your email address' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by danhauk** [comment]
👍 LGTM

**Pull request merged**